src/JWT/JWT.csproj

```
@@ -18,7 +18,7 @@
     <Authors>Alexander Batishchev, John Sheehan, Michael Lehenbauer</Authors>
     <PackageTags>jwt;json</PackageTags>
     <PackageLicense>CC0-1.0</PackageLicense>
-    <Version>8.1.5</Version>
+    <Version>8.2.0</Version>
```

This PR does not add a new feature. I would suggest to change only PATCH part of the version because it does not change the public API.

I think it is fine if user defines following in his csproj:
```xml
<PackageReference Include="JWT" Version="8.1.*" />
```
nuget resolves the optimized version.

```suggestion
    <Version>8.1.6</Version>
```

---

I just wanted to encourage users to pick up the updated version faster. Ok, we can downgrade it back.